web/controllers/user_category_controller.ex

```diff
@@ -45,4 +51,11 @@ defmodule CodeCorps.UserCategoryController do
     UserCategory |> Repo.get!(id) |> Repo.delete!
     conn |> send_resp(:no_content, "")
   end
+
+  defp authorize(conn, changeset) do
+    conn
+    |> assign(:changeset, changeset)
+    |> Canary.Plugs.authorize_resource(model: changeset)
+  end
+  defp authorized?(conn), do: conn |> Map.get(:assigns) |> Map.get(:authorized)
```

Comment:
Should these two methods be added to controller helpers since they are used in multiple controllers?